pyseir/inference/model_fitter.py

```diff
@@ -739,7 +767,7 @@ def run_state(state, states_only=False, with_age_structure=False):
 
         if len(all_fips) > 0:
             with Pool(maxtasksperchild=1) as p:
-                fitters = p.map(ModelFitter.run_for_fips, all_fips)
+                fitters = p.map(ModelFitter.run_for_region, map(RegionalInput.from_fips, all_fips))
```

Comment:
I think its a bit more clear to have the map on a separate line:
```
regions = [RegionalInput.from_fips(fips) for fips in all_fips]
fitters = p.map(ModelFitter.run_for_region, regions)

```

Reply:
Done